integration-tests/src/test/resources/features/PostcodeLookupRequest.feature

```diff
@@ -31,3 +31,15 @@ Feature: valid postcode test
     Examples:
       | testUserDataSheetRowNumber | testPostCode |
       | 197                        | SW1A 2AA     |
+
+  @postcode-lookup
+  Scenario Outline: postcode-lookup with too much request
```

Comment:
Hmm this outline isn't super clear. Do you think we could change it to something else like `Rate limit Is returned by OS API`?